package.json

```
@@ -55,16 +55,29 @@
   },
   "devDependencies": {
     "babelify": "^10.0.0",
+    "@mapbox/eslint-config-mapbox": "^3.0.0",
+    "@vercel/git-hooks": "^1.0.0",
     "browserify": "^17.0.0",
     "envify": "^4.1.0",
     "eslint": "^8.25.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "lint-staged": "^13.0.3",
+    "prettier": "^2.7.1",
     "tailwindcss": "^3.1.8",
     "tape": "~2.12.3"
   },
   "scripts": {
+    "lint": "eslint src/**/*",
+    "format": "prettier src/**/*",
```

This just prints a lot of formatted code for me but doesn't actually write. I think we want [the write option](https://prettier.io/docs/en/cli.html) enabled. This also does not pick up changes in `src/`, for example `src/ui.js`. I think we want `**` instead of `**/*`.

```suggestion
    "format": "prettier --write src/**",
```

---

fixed, nice catch